library/core/src/str/mod.rs

```diff
@@ -20,6 +20,19 @@ use crate::ops::Try;
 use crate::option;
 use crate::slice::{self, SliceIndex, Split as SliceSplit};
 
+#[allow(missing_docs)]
+#[stable(feature = "rust1", since = "1.0.0")]
+pub mod utf8;
+
+#[allow(missing_docs)]
+#[stable(feature = "rust1", since = "1.0.0")]
+pub mod indices;
```

Comment:
It'd be great if we could add doc comments for these modules.

Reply:
Would this be as straightfoward as converting the docs that were originally above the Utf8Error struct (and I guess the CharIndicies struct too) to docs for the module, or should I try write something else?

Reply:
To remove `allow(missing_docs)`, we should add module docs. In this case, I think it's okay to add a simple explanation e.g.:
https://github.com/rust-lang/rust/blob/5b04bbfcbb79ed7325ea2b580458a80d95da6bbb/library/core/src/borrow.rs#L1